app/src/main/java/com/kickstarter/libs/utils/ProjectViewUtils.kt

```diff
@@ -148,4 +148,41 @@ object ProjectViewUtils {
 
         return styledCurrency.trim()
     }
+
+    /**
+     * Returns a CharSequence representing a value in a project's currency based on a user's locale.
+     * The precision is shrunken and centered if the number is not whole.
+     * The currency symbol is shrunken and centered.
+     * Special case: US people looking at US currency just get the currency symbol.
+     */
+    fun styleCurrency(
```

Comment:
For compose UI the styling of the currency will happen on this ticket [https://kickstarter.atlassian.net/browse/MBL-1992](url). And using the new fields available on the API that already provide the currency and symbol. So for now ignore this new method here.